SwanContents/swancontents/filemanager/projects_mixin.py

```diff
@@ -204,16 +204,6 @@ def save(self, model, path=''):
 
         return model
 
-    def new(self, model=None, path=''):
```

Comment:
Why does `new` go away? We can just rely on the superclass' `new`? Why?

Reply:
Because in this commit we no longer use the type "project". Now it's a folder that contains a "project" attribute in the model. 

When Omar was here, we changed the logic only for the response. I simplified it even further to make the same logic in the creation thus removing extra logic required.